src/utils/handleRangeHeaders.js

```diff
@@ -1,6 +1,6 @@
 import parseRange from 'range-parser';
 
-export default function handleRangeHeaders(content, req, res) {
+export default function handleRangeHeaders(context, content, req, res) {
```

Comment:
Made a breaking change here because I think context should come first

Reply:
All is fine, we are in rc